ipamutils/utils.go

```diff
@@ -37,18 +37,37 @@ type NetworkToSplit struct {
 // InitNetworks initializes the broad network pool and the granular network pool
 func InitNetworks(defaultAddressPool []*NetworkToSplit) {
 	initNetworksOnce.Do(func() {
-		// error ingnored should never fail
-		PredefinedGranularNetworks, _ = splitNetworks(defaultGranularNetwork)
+		mutex.Lock()
+		// error ignored should never fail
+		PredefinedGlobalScopeDefaultNetworks, _ = splitNetworks(globalScopeDefaultNetworks)
+		mutex.Unlock()
 		if defaultAddressPool == nil {
```

Comment:
do you want to call this one localAddressPool?

Reply:
I guess its not required. Daemon Init will be called only once and we will be initializing local pool only one time during Init.  So I feel its not required.  WDYT?

Reply:
I think the point was more just for code clarity.  We are initializing two default address pools in this function.  Renaming the parameter makes it clear that one only can modify the local address pool using `InitNetworks()`.

Reply:
I agree, renaming the parameter to `localAddressPool` would make the intent clearer.